test/AutoDiff/forward_mode_runtime.swift

```diff
@@ -0,0 +1,30 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -Xllvm -run-jvp-generation -parse-stdlib -o %t/Builtins
+// RUN: %target-codesign %t/Builtins
+// RUN: %target-run %t/Builtins
+// REQUIRES: executable_test
+
+import Swift
+import StdlibUnittest
+
+var ForwardModeTests = TestSuite("ForwardMode")
+
+ForwardModeTests.test("Unary") {
+  func func_to_diff(x: Float) -> Float {
+    return x * x
+  }
+  let (y, differential) = Builtin.autodiffApply_jvp(func_to_diff, 4)
```

Comment:
Change this test to use differential operators instead.
```suggestion
  let (y, differential) = valueWithDifferential(at: 4, in: func_to_diff)
```

Simplify the RUN line and drop `import Swift` (which is relevant/necessary only with `-parse-stdlib`):
```
// RUN: %target-run-simple-swift
// REQUIRES: executable_test
```

Reply:
Offline discussion: will add a new lit command `%target_run_simple_swift_forward_mode_differentiation`